app/assets/stylesheets/responsive/custom.scss

```diff
@@ -1399,6 +1399,7 @@ p.locale-list-trigger {
   ul {
     list-style: none outside none;
     padding: 0;
+    li {padding: 5px 0;}
```

Comment:
Minor thing, but I’d format this onto a new line, like all of the other rules nearby:

```scss
  ul {
    list-style: none outside none;
    padding: 0;
    li {
      padding: 5px 0;
    }
  }
```